luminaire/model/window_density.py

```diff
@@ -546,7 +550,8 @@ def _get_result(self, input_df=None, detrend_order=None, agg_data_model=None, va
                 data_adjust_forecast = 1
             adjusted_execution_data = execution_data / data_adjust_forecast
             if detrend_order > 0:
-                adjusted_execution_data = adjusted_execution_data.to_list()
+                data_adjust_forecast = agg_data[-1][-1]
```

Comment:
Some logical code explanation would be helpful here

Reply:
added some comments